### ISSUE TITLE
Firefox cannot stream both video and audio

### ISSUE THREAD

**Issue by marcuiulian13** [feross/simple-peer#189]
Hi,

While developing an application using this library, during the final stages I realized that there is an issue with Firefox streaming.
The issue is that when streaming from / to FF, there is no way to make the browser send both audio and video with the stream (usually when both are requested, only audio is streamed). I've tested with sending just video or just audio and it works.
I've checked the `stream` object received from `getUserMedia` call and it contains a video track and an audio track, but when it reaches the other peer, it only contains the audio track.

L.E.:
For example, if I connect with a Chrome peer and stream video and audio, then i connect with Firefox and stream video and audio, the Chrome instance will get bot audio&video, but Firefox gets only the audio, so the issue seems to be that Firefox is not dealing well with both audio and video tracks.
To clarify that the issue is not Chrome, connecting Chrome-Chrome works fine.

Can someone help me out with this one? 
Is it a known issue? Is it a browser or a library issue, or is there something wrong with my implementation?

Thanks,
Iulian

**Comment by RationalCoding** [comment]
Just to be clear, you are trying to stream from Chrome->Firefox or Firefox->Firefox?

Could you also post OS versions, browser versions so that I can reproduce? Thanks.

**Comment by marcuiulian13** [comment]
Hi,
Here are some tests I did between FF and Chrome:
The initiator is always the left one (the first connected).
Both get each others sound in all test cases.

Firefox -> Firefox : neither gets the video
Firefox -> Chrome : Firefox gets Chrome video, but Chrome does not get video from Firefox
Chrome -> Firefox : Chrome gets Firefox video, but Firefox does not get video from Chrome
Chrome -> Chrome : both get video

Also another thing that might get us on track with what is the issue is that I found out that by debugging we find that on Firefox `Peer.prototype._onTrack` will get called twice: first with the audio track and the second time with the video track.

Thanks for help!
